tests/test_sync_bookmark.py

```diff
@@ -246,14 +246,38 @@ def test_validation_problems(test_transform):
                 }
             ]
         },
+        # new bookmarks engine with problems
+        {
+            'failure_reason': None,
+            'engines': [{
+                'name': 'bookmarks-buffered',
+                'validation': {
+                    'problems': [{
+                        'name': 'new problem',
+                        'count': 50,
+                    }, {
+                        'name': 'another problem',
+                        'count': 4,
+                    }],
+                },
+            }],
+        },
+        # new bookmarks engine without problems
+        {
+            'failure_reason': None,
+            'engines': [{
+                'name': 'bookmarks-buffered',
+                'validation': {'problems': None},
+            }],
+        },
     ])
 
-    assert df.count() == 4
+    assert df.count() == 6
 
     # one hot encode the cases
     assert df.select(
         F.sum('engine_validation_problem_count').alias('pcount')
-    ).first().pcount == 1111
+    ).first().pcount == 1165
 
```

Comment:
Can you add another assert that checks `engine_validation_problem_count` of `bookmark-buffered` is 54?